### PR TITLE
Display skulls on dead minions and heroes

### PR DESCRIPTION
**Pull request by Epix37**
This is based on the secrets PR to make my life a little easier. I'll rebase once that is merged.

Minions with 0 or less HP now have a skull icon to indicate that:
death_board

Even without actions, this makes it already a lot easier to see what's going on on the board.

Besides that, I looked into adding hero craters for the heroes. Turns out those are very large and the splash goes way on the board (won't look good in joust). So just adding the same icon to heroes seemed like the next best thing:
death_hero

If the hero was removed from play (and there is no new hero), the first from the graveyard will be used. 
Attack and armor icons are removed, health and secrets remain.

### CODE REVIEW

**Comment by jleclanche** [comment]
Awesome. And I'm down with keeping that skull on heroes instead of the crater I think.

File: ts/components/game/Player.tsx [hunk 1]
```diff
@@ -44,6 +44,9 @@ class Player extends React.Component<PlayerProps, {}> {
 
 		/* Equipment */
 		var heroEntity = playEntities.filter(filterByCardType(CardType.HERO)).first();
+		if (!heroEntity) {
+			heroEntity = (this.props.entities.get(Zone.GRAVEYARD) || Immutable.Map<number, Entity>()).first();
```
Comment (beheh) [inline ts/components/game/Player.tsx]: Not convinced by this yet. What happens with Jaraxxus/Ragnaros?

Comment (Epix37) [inline ts/components/game/Player.tsx]: You could view it as the hero "transforming back" after being defeated. I don't think it makes much of a difference either way.

Comment (jleclanche) [inline ts/components/game/Player.tsx]: I'm +1 on @Epix37's implementation.

Comment (Epix37) [inline ts/components/game/Player.tsx]: Oh I actually made a mistake here, this never filters for the hero. Lemme fix that...

**Comment by beheh** [comment]
Landed as 534e81467b931cec97206ce1582e673c92b4fbb0...9846a3799a23d6662aeb31098200453c795fed3d.

**Pull request closed without merging**